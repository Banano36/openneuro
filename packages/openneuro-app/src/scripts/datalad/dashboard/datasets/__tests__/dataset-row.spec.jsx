import React from 'react'
import { shallow } from 'enzyme'
import DatasetRow from '../dataset-row.jsx'

const dataset = {
  id: 'ds000001',
  draft: {
    description: {
      Name: 'test dataset',
    },
  },
}

describe('dashboard/datasets/DatasetRow', () => {
  it('renders successfully', () => {
    expect(shallow(<DatasetRow dataset={dataset} />)).toMatchSnapshot()
  })
})
