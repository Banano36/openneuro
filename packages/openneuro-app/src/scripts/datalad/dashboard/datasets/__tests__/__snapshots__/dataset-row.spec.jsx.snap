// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard/datasets/DatasetRow renders successfully 1`] = `
<div
  className="panel panel-default"
>
  <Styled(div)
    className="panel-heading"
  >
    <div
      className="header clearfix"
    >
      <Link
        to="/datasets/ds000001"
      >
        <h4
          className="dataset-name"
        >
          test dataset
        </h4>
        <Uploaded />
      </Link>
      <div
        className="metric-container"
      >
        <Metrics
          dataset={
            Object {
              "draft": Object {
                "description": Object {
                  "Name": "test dataset",
                },
              },
              "id": "ds000001",
              "snapshots": Array [],
            }
          }
          fromDashboard={true}
        />
      </div>
      <div
        className="status-container"
      >
        <ProfileComponent
          dataset={
            Object {
              "draft": Object {
                "description": Object {
                  "Name": "test dataset",
                },
              },
              "id": "ds000001",
              "snapshots": Array [],
            }
          }
          minimal={true}
        />
      </div>
    </div>
    <Summary
      minimal={true}
    />
  </Styled(div)>
</div>
`;
